content/en/continuous_integration/setup_tests/swift.md

```diff
@@ -205,6 +205,14 @@ For UITests, both the test target and the application running from the UITests m
 For the following configuration settings:
  - `Boolean` variables can use any of: `1`, `0`, `true`, `false`, `YES`, or `NO`
  - `String` list variables accept a list of elements separated by `,` or `;`
+ 
+### Enabling auto-instrumentation
+
+`DD_ENABLE_STDOUT_INSTRUMENTATION`
+: Captures messages written to `stdout` (e.g `print()` ) and reports them as Logs. (Implies charges for Logs product) (Boolean)
+
+`DD_ENABLE_STDERR_INSTRUMENTATION`
+: Captures messages written to `stderr` (e.g `NSLog()`, UITest steps ) and reports them as Logs. (Implies charges for Logs product) (Boolean)
```

Comment:
```suggestion
: Captures messages written to `stderr` (for example, `NSLog()`, UITest steps ) and reports them as logs. This may impact your bill. (Boolean)
```